.gitignore

```diff
@@ -90,7 +90,8 @@ target/
 #   However, in case of collaboration, if having platform-specific dependencies or dependencies
 #   having no cross-platform support, pipenv may install dependencies that don't work, or not
 #   install all needed dependencies.
-#Pipfile.lock
+Pipfile
+Pipfile.lock
```

Comment:
What's the reason for not committing the dependency lock file? I would only *not* do this if there is a problem...

Reply:
As we use requirements.txt and dev_requirements.txt for listing dependencies a Pipfile would duplicate the requirements.